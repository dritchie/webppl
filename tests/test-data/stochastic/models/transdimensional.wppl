var model = function() {
	var a = flip(0.9) ? beta(1, 5) : 0.7;
	var b = flip(a);
	factor(b ? 0 : -Infinity);
	return a;
};