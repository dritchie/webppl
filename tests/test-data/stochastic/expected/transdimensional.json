{
  "mean": 0.417
}